Code/Framework/AzAndroid/java/com/amazon/lumberyard/LumberyardActivity.java

```diff
@@ -64,7 +64,7 @@ public void onBackPressed()
 
     ////////////////////////////////////////////////////////////////
     // called from the native to get the application package name
-    // e.g. com.lumberyard.samples for SamplesProject
+    // e.g. com.o3de.samples for SamplesProject
```

Comment:
We should replace all `com.o3de.samples` with `org.o3de.samples` to reflect `03de.org`, (https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html )

Reply:
Done.